packages/hdwallet-native/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@shapeshiftoss/hdwallet-core": "^1.11.1",
+    "@zxing/text-encoding": "^0.9.0",
```

Comment:
This is an optional dependency of `web-encoding`, and provides `TextEncoder` support when it's not available natively. It's a fork of the now-unmaintained `text-encoding` package.